include/envoy/request_info/request_info.h

```diff
@@ -216,7 +223,12 @@ class RequestInfo {
   /**
    * @return whether response flag is set or not.
    */
-  virtual bool getResponseFlag(ResponseFlag response_flag) const PURE;
+  virtual bool hasResponseFlag(ResponseFlag response_flag) const PURE;
+
+  /**
+   * @return whether any response flag is set or not.
+   */
+  virtual bool hasResponseFlag() const PURE;
```

Comment:
IMO this function name is a little strange given the other overload of same name Can we call it `hasAnyResponseFlag()` or something similar?